apps/frontend/src/libs/components/header/styles.module.css

```
@@ -0,0 +1,28 @@
+.header {
+	box-sizing: border-box;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	width: 100%;
+	padding: 12px 32px 12px 48px;
```

Hm, why would header have different paddings on left/right? If you were trying to account for space in a user info, this padding should be applied to the user info container itself
```suggestion
	padding: 12px 32px;
```